packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -37,23 +37,26 @@ This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aw
 
 ## Table of Contents
 
-- [User Pools](#user-pools)
-  - [Sign Up](#sign-up)
-  - [Sign In](#sign-in)
-  - [Attributes](#attributes)
-  - [Security](#security)
-    - [Multi-factor Authentication](#multi-factor-authentication-mfa)
-    - [Account Recovery Settings](#account-recovery-settings)
-  - [Emails](#emails)
-  - [Device Tracking](#device-tracking)
-  - [Lambda Triggers](#lambda-triggers)
-    - [Trigger Permissions](#trigger-permissions)
-  - [Import](#importing-user-pools)
-  - [Identity Providers](#identity-providers)
-  - [App Clients](#app-clients)
-  - [Resource Servers](#resource-servers)
-  - [Domains](#domains)
-  - [Deletion protection](#deletion-protection)
+- [Amazon Cognito Construct Library](#amazon-cognito-construct-library)
+  - [Table of Contents](#table-of-contents)
+  - [User Pools](#user-pools)
+    - [Sign Up](#sign-up)
+    - [Sign In](#sign-in)
+    - [Attributes](#attributes)
+    - [Attribute verification](#attribute-verification)
+    - [Security](#security)
+      - [Multi-factor Authentication (MFA)](#multi-factor-authentication-mfa)
+      - [Account Recovery Settings](#account-recovery-settings)
+    - [Emails](#emails)
+    - [Device Tracking](#device-tracking)
+    - [Lambda Triggers](#lambda-triggers)
+      - [Trigger Permissions](#trigger-permissions)
+    - [Importing User Pools](#importing-user-pools)
+    - [Identity Providers](#identity-providers)
+    - [App Clients](#app-clients)
+    - [Resource Servers](#resource-servers)
+    - [Domains](#domains)
+    - [Deletion protection](#deletion-protection)
```

Comment:
Why...? Can leave this as it was?

Reply:
I'm sorry... My local VSCode formatter was formatting the contents of the README.
This is not related to this PR, so I have corrected it back.